int32 instance_id
uint32 class_id
float32 confidence
